src/Aquifer.Data/Schemas/ParentResourceLicenseInfoSchema.cs

```diff
@@ -1,6 +1,6 @@
-﻿namespace Aquifer.API.Endpoints.Resources;
+namespace Aquifer.Data.Schemas;
 
-public sealed class ResourceLicenseInfo
+public sealed class ParentResourceLicenseInfoSchema
```

Comment:
Technically this will break the auto-generated Public API client if anyone is using the old `ResourceLicenseInfo` generated class but that's probably fine.

But see my thoughts above about dropping `Schema`.